ftw/theming/resources/scss/elements/form.scss

```diff
@@ -0,0 +1,42 @@
+$color-input: $color-white !default;
+$line-height-input: $line-height-base !default;
+$border-color-input: $color-gray-dark !default;
+$font-size-input: 1em !default;
+$color-input-active: $color-primary !default;
+
+@mixin input() {
+  position: relative;
+  font-weight: normal;
+  font-style: normal;
+  font-size: $font-size-input;
+  @include auto-text-color($color-input);
+  margin: 0;
+  max-width: 100%;
+  outline: none;
+  line-height: $line-height-input;
+
```

Comment:
Can we increase the default input width?
It is very short at the moment.
![bildschirmfoto 2016-01-29 um 11 52 57](https://cloud.githubusercontent.com/assets/7469/12673768/fef9cefc-c67e-11e5-9056-4df83a9c2bc1.png)